server/lib/init_log.rb

```diff
@@ -0,0 +1,24 @@
+require "fusor/multilog"
+
+module Fusor
+  @default_log_file = "/var/log/foreman/fusor.log"
+
+  def self.log
+    @log ||= MultiLogger.new(Rails.logger)
+  end
+
+  def self.log_change_deployment(deployment = nil)
+    self.log
+
+    #@log.detach_all
+
+    if Rails.env.production?
```

Comment:
I would advise against this. This can cause errors to go undetected in development/test. If you're trying to get this to work in Travis/test environment, it'd be better to mock or stub this method. Or even better, use [FakeFS](https://github.com/defunkt/fakefs) which foreman provides.